app/models/list_set.rb

```diff
@@ -40,31 +40,29 @@ def lists
   def a_to_z_list
     [ListSet::List.new(
       "A to Z",
-      content_tagged_to_tag.reject do |content|
-        excluded_formats.include? content.format
-      end.sort_by(&:title)
+      content_tagged_to_tag
+        .reject { |content| excluded_formats.include? content.format }
+        .sort_by(&:title)
     )]
   end
 
   def curated_list
-    @group_data.map do |group|
-      contents = group["contents"].map do |base_path|
-        content_tagged_to_tag.find do |content|
+    @group_data.map { |group|
+      contents = group["contents"].map { |base_path|
+        content_tagged_to_tag.find { |content|
           content.base_path == base_path
-        end
-      end.compact
+        }
+      }.compact
 
       ListSet::List.new(group["name"], contents) if contents.any?
-    end.compact
+    }.compact
   end
 
   def content_tagged_to_tag
-    @content_tagged_to_tag ||= RummagerSearch.new({
-      :start => 0,
+    @content_tagged_to_tag ||= RummagerSearch.new(:start => 0,
```

Comment:
I've moved `:start` to another line.

I've left the key syntax as-is because one of the keys is generated dynamically (line 65), so we're forced to use the `=>` syntax:

```
filter_name => [@tag_content_id],
```

```filter_name``` is actually a method call, because the filter name depends on the tag type.

I might have overlooked a different way to solve this, though. :)